conversion/conversion.go

```diff
@@ -676,12 +676,15 @@ func CreateOrUpdateDatabase(ctx context.Context, adminClient *database.DatabaseA
 		ctx = metadata.AppendToOutgoingContext(ctx, constants.MigrationMetadataKey, migrationMetadataValue)
 	}
 	if dbExists {
+		if conv.SpDialect != constants.DIALECT_POSTGRESQL && migrationType == constants.DATAFLOW_MIGRATION {
+			return fmt.Errorf("Low downtime schema migration cannot work on existing database")
```

Comment:
Lets reword this to "Harbourbridge does not support minimal downtime schema/schema-and-data migrations to an existing database".

Reply:
Done